### DISCUSSION TITLE
Popover2 backdrop animated appearance doesn't work

### DISCUSSION THREAD

**Discussion by sashaglobenko** [palantir/blueprint#5873] [Q&A] [answered]
#### Environment

- 3.47.0
- mac os
- Chrome 92.0.4515.131

#### Code Sandbox

https://codesandbox.io/s/blueprint-sandbox-forked-w6qej?file=/src/Popover2Example.tsx

#### Steps to reproduce

1. Click button

#### Actual behavior

Backdrop appears without any animation like when open Dialog.

#### Expected behavior

Backdrop appears with fade animation

**Comment by adidahiya** [comment, marked as answer]
Are you trying to add a custom popover backdrop CSS transition?

1. Why?
2. This should be doable, but you'll need to get a little more specific / in the weeds with the TransitionGroup classes... you probably need to attach a custom class to your backdrop and then target it with `.my-popover-backdrop.bp3-popover2-enter { ... }` ([see CSSTransition docs](http://reactcommunity.org/react-transition-group/css-transition))
